lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -258,6 +258,37 @@ bool Ohmbrewer::Thermostat::isOff() const {
  * Performs the Thermostat's current task. Expect to use this during loop().
  * This function is called by work().
  *
+ *
+ * @returns The time taken to run the method
+ */
+int Ohmbrewer::Thermostat::doWork() {
+    unsigned long start = micros();
+
+    if (getState()){
+        //enable timer
+//        _timer->start();
+        doPID();
+    }else{
+        //Shut down procedure
+        //if thermostat is turned off then turn off element too.
+        getElement()->setState(false);
+        getElement()->work();//reset element
+        //stop Timer
+//        _timer->stop();
+        //if PID is off then update temp sensor
+        getSensor()->work();
```

Comment:
You should shut the sensor off too. Otherwise it's going to keep sending temperature updates.